docs/CONTAINER_INTERFACE.md

```diff
@@ -26,14 +26,17 @@ manager, please consider supporting the following interfaces.
    to prevent the container from altering the host kernel's configuration
    settings. (As a special exception, if your container has network namespaces
```

Comment:
I would remove the '(' here. The paragraph is so long that the parentheses just become confusing